etc/DependencyInstaller.sh

```diff
@@ -69,6 +69,16 @@ _installCommonDev() {
         echo "Eigen already installed."
     fi
 
+    # CUSP
+    if [[ ! -d /usr/local/include/cusp/ ]]; then
+        cd "${baseDir}"
+        git clone -b cuda9 https://github.com/cusplibrary/cusplibrary.git
+        cd cusplibrary
+        sudo cp -r ./cusp /usr/local/include/
```

Comment:
@vvbandeira any concern about the use of sudo?

Reply:
Is not necessary. The script requires to be run as root or with sudo, otherwise it will fail on apt/yum commands.